tdigest/merging_digest.go

```diff
@@ -36,8 +36,7 @@ type MergingDigest struct {
 	min           float64
 	max           float64
 	reciprocalSum float64
-
-	debug bool
```

Comment:
Why rip all this out instead of just not using it?

Reply:
It's a pretty small point, but int64 * every single centroid * ever single histogram in the smart gateway. It's 60-120mb per smart gateway. /shrug.

Reply:
Ah, great! (This would've made great fodder for the PR body. :smile:)

Reply:
Fair point.